### ISSUE TITLE
How to setup CORS settings

### ISSUE THREAD

**Issue by meta-forte** [jacekschae/learn-reagent-course-files#20]
I am trying to follow the reagent videos. The Gigs Image URLs are not loading due to CORS errors. Please advise where to configure CORS settings

**Comment by jacekschae** [comment]
This depends on the browser you are using. Normally this would be done with an extension.

**Comment by meta-forte** [comment]
Auto resolved on its own. Not sure why it failed in the first place. Thank you very much for the quick response.

**Comment by chase-lambert** [comment]
I was having a CORS issue with the Google avatar not showing up after login. So sort of related to this one. I was able to solve it by adding a `:referrerpolicy "no-referrer"` attribute to the image element like so: 

`     [:img {:src (:photo-url @state/user)                                                      
              :referrerpolicy "no-referrer"}]]]`
 
Figured I would throw this here in case anyone ends up having similar issues. I got the information from: https://stackoverflow.com/questions/40570117/http403-forbidden-error-when-trying-to-load-img-src-with-google-profile-pic
